testing/test_minibatch_sequential_pipeline.py

```diff
@@ -72,7 +72,7 @@ def test_mini_batch_sequential_pipeline_should_fit_transform_steps_sequentially_
     ])
 
     # When
-    p, outputs = p.fit_transform(range(20), range(20))
+    p, outputs = p.fit_transform(list(range(20)), list(range(20)))
```

Comment:
Hey this is a cool test. Like what would happen if `list` casts were removed, could it still work ? I think this would be the object of an issue for further review later.

Reply:
yes it would still work. however, I prefered having list instead of ranges for my assertions below